CMSgov/eAPD
# Tforkner/4060 metrics key

Resolves #4060 

### Description
Updated apd loader to assign keys to metrics.

### Steps to manually verify this change

1. open an existing APD
2. look at the reducer in the browser and verify that outcomes looks the same as before and metrics now has keys

### This pull request is ready to review when


### This pull request can be merged when
